javageci-core/src/main/java/javax0/geci/equals/Equals.java

```diff
@@ -93,6 +93,7 @@ private void generateEqualsHeader(Segment segment, Class<?> klass, CompoundParam
         segment.write("@Override")
                 .write_r("public %sboolean equals(Object o) {", subclassingAllowed ? "final " : "")
                 .write("if (this == o) return true;");
+        segment.write("if (!super.equals(o)) return false;");
```

Comment:
With this modification, all class that has the `Object` as a parent will become equal only when `a == b` the same object.

Must be configurable and must be used only if there is a parent class other than `Object` and that class defines `equals()` or inherits from some other parent but not from `Object`.